_posts/2022-09-27-call-for-python-software-review-editors.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: single
+title: "pyOpenSci Editorial Process - New Editor in Chief and a Call For Editors"
+excerpt: "pyOpenSci is developing it's open peer review process for Python scientific software. Learn about our structure and if you are interested, apply to be on our editorial board."
+author: "Leah Wasser"
+permalink: /blog/python-software-review-process-editor-in-chief
+header:
+    overlay_color: "#666"
+    overlay_filter: 0.6
+categories:
+  - updates
+  - blog-post
+  - highlight
+  - peer-review
+---
+
+## An update on pyOpenSci's review process 
+
+Hey there! I just wanted to update everyone about where pyOpenSci is with 
+it's peer review process. We are currently hard at work updating [our peer 
+review guides](https://www.pyopensci.org/contributing-guide/intro.html) to 
+streamline the peer review process. For the next 2 months (October & November 2022) 
+we will prioritize setting up a editorial board to support new reviews as they 
+come in. 
+
+With that below you will find a few updates.
+
+## Welcome Editor in Chief David Nicholson 
+
+[David Nicholson](https://github.com/NickleDave), who has been working with pyOpenSci for the past few years as 
+an editor will take on the Editor in Chief role! [You can learn more about that 
+role here](https://www.pyopensci.org/contributing-guide/open-source-software-submissions/editor-in-chief-guide.html) in our peer review guide. David will:
+
+* Be the first person to reach out to anyone submitting a new review or a pre-submission. 
+* He will perform initial software checks to ensure your package is ready for  
+* He will also facilitate kicking off the peer review process by finding an editor for your package
+
+David has a background behavioral / neuro / cognitive science and is a ML guru. 
+Of course,  he's a Pythonista thru and thru. We are lucky to have David as a 
+part of our pyOpenSci team. 
+
+Stay tuned for a blog post from David introducing himself.
+
+## We are creating an editorial board!
+
+To support our open peer review, we want to bring on a few people (like yourself?!) 
+to serve as guest editors / editors. Here's how this works:
+
+Editors at pyOpenSci make a minimum of a 1-year commitment with a possibility
+of longer given mutual consent. We also 
+support guest editors in some instances. You will be responsible for handling 3-4 
+package reviews during that year. If you are new to our organization, we start 
```

Comment:
The logical flow in this paragraph jumps around a bit. Maybe try this way?

> If you are new to our organization, we start you as a guest editor. What this means is that you will review one package and we will support you in that process to ensure you learn the ropes. Then if you are happy and work well with our editorial team we will bring you on officially as an editor. Editors at pyOpenSci make a minimum of a 1-year commitment with a possibility of longer given mutual consent. You will be responsible for handling 3-4 package reviews during that year. We may also bring on guest editors in other instances, e.g., because of their familiarity with a specific domain.

Reply:
added!! psss you can highlight multiple lines in a document and comment as well :) but i fixed this locally :)